Plugins/Public/base_plugin/Main.cpp

```diff
@@ -829,6 +830,20 @@ void HkTimerCheckKick()
 		// Dispatch timer but we can safely ignore the return
 		base->Timer(curr_time);
 	}
+	if (!player_bases.empty() && !set_holiday_mode) {
+		if (baseSaveIterator == player_bases.end()) {
+			baseSaveIterator = player_bases.begin();
+		}
+		bool saveSuccessful = false;
+		while (!saveSuccessful && baseSaveIterator != player_bases.end()) {
+			auto& pb = baseSaveIterator->second;
+			if (pb->logic == 1 || pb->invulnerable == 0) {
```

Comment:
where does this criteria come from?

Reply:
We only need to open and save to a POB file if it has a chance to change. It means that it needs to either have passive logic enabled (passive damage, factories, healing etc) or it needs to be able to take damage.

Basically it excludes indestructible POBs that have no logic attached to them, such as nomad POB JHs in Omicron Iota.